deploy.sh

```diff
@@ -44,37 +44,35 @@ ${bold}EXAMPLE${normal}
   exit
 fi
 
-
-length=$(($#-1))
-OPTIONS=${@:1:$length}
-REPONAME="${!#}"
 CWD=$PWD
+case $1 in
+  -t=*|--target=*)
+    CWD="$CWD/${1#*=}"
+    shift
+  ;;
+  *)
+  ;;
+esac
 
-for i in "${@:2}"
-do
-  case $i in
-    -t=*|--target=*)
-      CWD="${i#*=}"
-    ;;
-    *)
-    ;;
-  esac
-done
-
-LIBRARIES=()
-DEPENDENCIES=()
-PROYECTS=()
```

Comment:
Antes tenía un for loop que iba desde el segundo parámetro, es decir que en vez de tomar solo el primero, lo evitaba :P